cvpy/image/NaturalImageTable.py

```diff
@@ -1,21 +1,192 @@
-from typing import List, Dict
+#!/usr/bin/env python
+# encoding: utf-8
+#
+# Copyright SAS Institute
+#
+#  Licensed under the Apache License, Version 2.0 (the License);
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+#
 
-from swat import CASTable
+''' Image analysis tools '''
 
+import sys
+import struct
+import numpy as np
+from typing import List, Dict
+from warnings import warn
+from swat import CASTable
 from cvpy.base.ImageTable import ImageTable
+from cvpy.utils.RandomNameGenerator import RandomNameGenerator
+from enum import *
 
 
 class NaturalImageTable(ImageTable):
+    """
+    Implement image processing functions.
+
+     Parameters
+     ----------
```

Comment:
Update the documentation for the parameters following the parameters added to the constructor.